modules/core/04-channel/keeper/packet.go

```diff
@@ -67,7 +67,7 @@ func (k Keeper) SendPacket(
 
 	clientState, found := k.clientKeeper.GetClientState(ctx, connectionEnd.GetClientID())
 	if !found {
-		return 0, clienttypes.ErrConsensusStateNotFound
+		return 0, clienttypes.ErrClientNotFound
```

Comment:
This is state machine breaking :eyes: since the code is changing. Can we make the change in a separate pr in case this gets backported?

Reply:
Yup, the fix for the CLI is intended for `v7.1.0` which will require a co-ordinated upgrade anyway. But I agree with pulling out this error change to a separate PR 👍

Reply:
https://github.com/cosmos/ibc-go/pull/3593